src/features/courses/course-controller.ts

```diff
@@ -311,23 +311,27 @@ class CourseController {
                 }
                 const curriculum = await curriculumRepository.getCurriculumById(options.object.curriculumId);
                 const createdCourse = await courseRepository.createCourse(options.object);
+                logger.debug('Created a course from a curriculum.');
                 await curriculum.units?.asyncForEach(async (curriculumUnit: CurriculumUnitContent) => {
                     if (curriculumUnit.active === false) {
-                        logger.warn(`Inactive curriculum unit was fetched in query for create course ID#${curriculumUnit.id}`);
+                        logger.debug(`Inactive curriculum unit was fetched in query for create course ID#${curriculumUnit.id}`);
```

Comment:
I guess this should be a debug because it should fail if it happens and the frontend should report the issue
There is really nothing wrong with the backend here